app/views/application/info_pair_programming.html.slim

```diff
@@ -0,0 +1,7 @@
+= render_markdown(t('programming_groups.new.info_pair_programming'))
+= render_markdown(t('programming_groups.new.info_study'))
+== t('exercises.implement.pair_programming_feedback', url: 'https://etherpad.xopic.de/p/openHPI_PairProgrammingFeedback')
+
+- content_for :modal_footer do
+  button.btn.btn-primary data-bs-dismiss= 'modal' onclick= 'CodeOceanEditor.setStoredViewedPPInfo(true);'
+    = t('programming_groups.new.dont_show_modal_again')
```

Comment:
My proposal would be to have two buttons on this modal: One to close it (blue, primary), and another one to hide it on this device (potentially just gray, secondary). This way, learners might not close it unintended...